packages/e2e-test-app-fabric/windows/RNTesterApp-Fabric/RNTesterApp-Fabric.cpp

```diff
@@ -0,0 +1,283 @@
+    host.InstanceSettings().UseFastRefresh(m_fastRefreshEnabled);
+    host.InstanceSettings().DebuggerPort(m_debuggerPort);
+    host.InstanceSettings().UseDeveloperSupport(true);
+
+    host.PackageProviders().Append(winrt::make<CompReactPackageProvider>());
+    winrt::Microsoft::ReactNative::ReactCoreInjection::SetTopLevelWindowId(
+        host.InstanceSettings().Properties(), reinterpret_cast<uint64_t>(hwnd));
+
+    // Nudge the ReactNativeHost to create the instance and wrapping context
+    host.ReloadInstance();
+
+    winrt::Microsoft::ReactNative::ReactViewOptions viewOptions;
+    viewOptions.ComponentName(appName);
+    m_CompositionHwndHost.ReactViewHost(
+        winrt::Microsoft::ReactNative::ReactCoreInjection::MakeViewHost(host, viewOptions));
+
+    auto windowData = WindowData::GetFromWindow(hwnd);
+    if (!windowData->m_windowInited) {
+      m_CompositionHwndHost.Initialize(static_cast<uint64_t>(hwnd));
+      windowData->m_windowInited = true;
+    }
+    return 0;
+  }
+
+  LRESULT TranslateMessage(UINT message, WPARAM wparam, LPARAM lparam) noexcept {
+    if (m_CompositionHwndHost) {
+      return static_cast<LRESULT>(m_CompositionHwndHost.TranslateMessage(message, wparam, lparam));
+    }
+    return 0;
+  }
+};
+
+extern "C" IMAGE_DOS_HEADER __ImageBase;
+HINSTANCE WindowData::s_instance = reinterpret_cast<HINSTANCE>(&__ImageBase);
+
+//
+//  FUNCTION: WndProc(HWND, UINT, WPARAM, LPARAM)
+//
+//  PURPOSE: Processes messages for the main window.
+//
+LRESULT CALLBACK WndProc(HWND hWnd, UINT message, WPARAM wParam, LPARAM lParam) {
+  auto windowData = WindowData::GetFromWindow(hWnd);
+  if (windowData) {
+    auto result = WindowData::GetFromWindow(hWnd)->TranslateMessage(message, wParam, lParam);
+    if (result)
+      return result;
+  }
+
+  switch (message) {
+    case WM_DESTROY: {
+      delete WindowData::GetFromWindow(hWnd);
+      SetProp(hWnd, WindowDataProperty, 0);
+      PostQuitMessage(0);
+      return 0;
+    }
+    case WM_NCCREATE: {
+      auto cs = reinterpret_cast<CREATESTRUCT *>(lParam);
+      auto windowData = static_cast<WindowData *>(cs->lpCreateParams);
+      WINRT_ASSERT(windowData);
+      SetProp(hWnd, WindowDataProperty, reinterpret_cast<HANDLE>(windowData));
+      break;
+    }
+    case WM_GETOBJECT: {
+      if (lParam == UiaRootObjectId) {
+        auto windowData = WindowData::GetFromWindow(hWnd);
+        if (!windowData->m_windowInited)
+          break;
+
+        auto hwndHost = windowData->m_CompositionHwndHost;
+        winrt::com_ptr<IRawElementProviderSimple> spReps;
+        hwndHost.UiaProvider().as(spReps);
+        LRESULT lResult = UiaReturnRawElementProvider(hWnd, wParam, lParam, spReps.get());
+        return lResult;
+      }
+    }
+  }
+
+  return DefWindowProc(hWnd, message, wParam, lParam);
+}
+
+int RunRNTester(int showCmd) {
+  constexpr PCWSTR appName = L"React Native Tester (Composition)";
```

Comment:
Not the same as what's defined on line 117:
```
PCWSTR appName = L"RNTesterApp";
```

These will probably want to move out when this becomes an app template anyway. Put them as consts higher in the file?